### DISCUSSION TITLE
Connecting to a device | ShellHub

### DISCUSSION THREAD

**Discussion by giscus** [shellhub-io/shellhub#3234] [General]
# Connecting to a device | ShellHub

You can establish an SSH connection to a device from the Devices page.

https://docs.shellhub.io/user-guides/devices/adding

**Comment by osvaldolopes** [comment]
I'm still testing the tool, with servers running truenas can I get ssh access?

**Reply by henrybarreto** [reply]
Hi, @DicoLOF.

We have created a FreeBSD port for our Agent. 

https://github.com/shellhub-io/ports
